### PR TITLE
Testing accept comment feature on main

### PR DESCRIPTION
**Pull request by jeana-redhat**
This is a test PR. More info [here](https://docs.google.com/document/d/1Gm7SRwD4Q_pJe5QSDNdG6Y1bIReD-W3OO4adL9PoJ3s/edit).

**Conditions**
Base branch: `main`
Origin updated: No
Checkout against: `upstream/main`

### CODE REVIEW

File: machine_management/index-test.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+[source,terminal]
+----
+Nunc dapibus facilisis elit, quis sodales tortor rutrum dapibus.
+----
+
+[IMPORTANT]
+====
+Suspendisse quis dictum ipsum.
+In id neque et ligula egestas interdum.
+Morbi convallis vitae velit eget pulvinar.
```
Comment (maxwelldb) [inline machine_management/index-test.adoc:40]:
```suggestion
Rōma in Italiā est.
Italia in Eurōpā est. 
Graecia in Eurōpā est.
```

File: machine_management/index-test.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+= Test file to mess with for DOL
+include::_attributes/common-attributes.adoc[]
+:context: index-test-gh-comments
+
+toc::[]
+
+Here is a file to mess with to try to accept comments directly via GH review features.
+Text from the link:https://www.lipsum.com[Lorem Ipsum] generator.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean consectetur placerat purus vel pulvinar. Ut id dignissim erat. Nam volutpat consequat posuere. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce massa dolor, interdum egestas imperdiet non, accumsan id lorem. Sed sed ultrices quam, nec aliquet orci. Cras sodales ac quam id hendrerit. Vivamus et eleifend ipsum, sed iaculis lorem. Nulla scelerisque, ex a dapibus dignissim, ipsum libero hendrerit neque, ac scelerisque odio sapien quis ipsum. Maecenas in hendrerit risus. Nullam eget sapien massa. Donec magna quam, vehicula eu elementum at, volutpat et est. Cras vel suscipit nisi, aliquam congue mauris. In fermentum nulla sed tortor imperdiet tristique.
```
Comment (ShaunaDiaz) [inline machine_management/index-test.adoc:12]: Verba ita sunt intelligenda ut res magis valeat quam pereat.

File: machine_management/index-test.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index-test-gh-comments"]
+= Test file to mess with for DOL
+include::_attributes/common-attributes.adoc[]
+:context: index-test-gh-comments
+
+toc::[]
+
+Here is a file to mess with to try to accept comments directly via GH review features.
```
Comment (snarayan-redhat) [inline machine_management/index-test.adoc:9]:
```suggestion
Here is a file to mess with to try to accept comments directly via github review features.
```

File: machine_management/index-test.adoc [hunk 4]
```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index-test-gh-comments"]
+= Test file to mess with for DOL
```
Comment (snarayan-redhat) [inline machine_management/index-test.adoc:3]:
```suggestion
= Test file to mess with for DOL!!!
```

File: machine_management/index-test.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+----
+
+[IMPORTANT]
+====
+Suspendisse quis dictum ipsum.
+In id neque et ligula egestas interdum.
+Morbi convallis vitae velit eget pulvinar.
+====
+
+Pellentesque semper suscipit lorem, vitae dapibus metus suscipit posuere. Fusce lobortis eros ut urna dignissim, ac porttitor ipsum vulputate. Etiam sed enim erat. Vestibulum quis nisi non erat pharetra elementum. Morbi eu ex felis. Aenean quis massa dolor. Integer mollis pulvinar est, ut gravida orci sollicitudin sed. Sed eget venenatis mauris. Morbi faucibus est eu mi hendrerit faucibus. Morbi consequat mauris nec pellentesque dignissim. Nullam tempor malesuada neque. Nulla quis est quis risus tempor dapibus vel ac ipsum.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply machine_management/index-test.adoc:43]: It automatically translates these in my email. Hilarious and bizarre.

File: machine_management/index-test.adoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index-test-gh-comments"]
+= Velle est posse
+include::_attributes/common-attributes.adoc[]
+:context: index-test-gh-comments
+
+toc::[]
+
+Here is a file to mess with to try to accept comments directly via github review features.
+Text from the link:https://www.lipsum.com[Lorem Ipsum] generator.
```
Comment (jeana-redhat) [inline machine_management/index-test.adoc:10]:
```suggestion
Text from the link:https://www.lipsum.com[Lorem Ipsum] generator!
```

**Pull request closed without merging**